uefi-mkconfig

```diff
@@ -115,7 +115,7 @@ main () {
 	elif [[ -z ${mounted_efi_partitions} ]]; then
 		die "No mounted efi partitions!"
 
-	elif [[ ! -x "$(which efibootmgr)" ]]; then
+	elif [[ ! -x "$(command -v efibootmgr)" ]]; then
 		die "efibootmgr dependency not found!"
```

Comment:
Sorry, but "dependency" may mislead people, they can assume there's no required package installed while in fact it can be simply missing from screwed "path" variable. Maybe might be reasonable to print that command is not found in PATH locations?..

Reply:
Good point, I agree